Gemfile.lock

```diff
@@ -496,7 +496,7 @@ GEM
     mini_portile2 (2.3.0)
     minitar (0.6.1)
     minitest (5.11.3)
-    mixlib-archive (0.4.16)
+    mixlib-archive (0.4.18)
```

Comment:
I think you want to hold off on this for now. I need to sort out some windows stuff

Reply:
Bundle update should fix this now